endpoints/openrtb2/amp_auction_test.go

```diff
@@ -4,24 +4,66 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/exchange"
 	"github.com/prebid/prebid-server/pbsmetrics"
 	"github.com/prebid/prebid-server/stored_requests/backends/empty_fetcher"
 	"github.com/rcrowley/go-metrics"
-	"net/http"
-	"net/http/httptest"
-	"testing"
 )
 
 // From auction_test.go
 // const maxSize = 1024 * 256
 
 // TestGoodRequests makes sure that the auction runs properly-formatted stored bids correctly.
 func TestGoodAmpRequests(t *testing.T) {
+	testAmpStoredRequestData := map[string]json.RawMessage{
+		"10":  json.RawMessage(validRequest(t, "site.json")),
+		"11":  json.RawMessage(validRequest(t, "app.json")),
+		"12":  json.RawMessage(validRequest(t, "timeout.json")),
+		"100": json.RawMessage("5"),
+		"101": json.RawMessage("5"),
```

Comment:
Ok, that was a bug in the original AMP tests, should have been testing all the invalid requests. Not terribly important though, since this shares code with openrtb, and the invalid requests should be handled there anyway. I had failed to advance the index in `invalidRequests[]` on each line.

Reply:
ah... I see. Yeah, I didn't pay much attention to the semantics of these tests, but I'll clean them up so they make sense again.